website/src/pages/documents/document.css.ts

```diff
@@ -224,3 +225,18 @@ export const mobileWordPanel = style([
     },
   },
 ])
+
+export const scrollTop = style({
```

Comment:
In here, a lot of your styles are overriding each other and you can remove some, such as your `flexDirection` and `flexWrap` since the `flexFlow` overrides this.